watertap/costing/watertap_costing_package.py

```diff
@@ -793,6 +868,61 @@ def cost_membrane(blk, membrane_cost, factor_membrane_replacement):
     )
 
 
+def cost_electrodialysis_stack(
+    blk,
+    membrane_cost,
+    spacer_cost,
+    membrane_replacement_factor,
+    electrode_cost,
+    electrode_replacement_factor,
+):
+    """
+    Generic function for costing the stack in an electrodialysis unit.
+    Assumes the unit_model has a `cell_pair_num`, `cell_width`, and `cell_length`
+    set of variables used to size the total membrane area.
+
+    Args:
+        membrane_cost - The total cost of the CEM and AEM per cell pair in currency per area
+        spacer_cost - The total cost of the spacers per cell pair in currency per area
+        membrane_replacement_factor - Replacement factor for membranes and spacers
+                                      [fraction of membranes/spacers replaced/year]
+        electrode_cost - The total cost of electrodes in a given stack in currency per area
+        electrode_replacement_factor - Replacement factor for electrodes
+                                        [fraction of electrodes replaced/year]
+
+    """
+

```

Comment:
If this function were to be used as a stand-alone, *it* should call `make_capital_cost_var` and `make_fixed_operating_cost_var`, as opposed to requiring these vars to be created first.